app/assets/stylesheets/_navbar.scss

```diff
@@ -181,3 +146,35 @@
     margin-right: 0;
   }
 }
+
+.locale {
```

Comment:
I was using it just to compose the classes names. 
```
.locale {
  &-wrapper { ... }
  &-select { ... }
}
```
What do you think if I just separate them? it would make it clearer?
```
.locale-wrapper { .. }
.locale-select { ... }
```